sale_stock_invoice_lot_quantity/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_lot_quantity,lot_quantity,sale_stock_invoice_lot_quantity.model_lot_quantity,base.group_user,1,1,1,1
```

Comment:
Please double-check if this is correct, considering that the report may be generated by portal users.